client/packages.go

```diff
@@ -117,3 +118,27 @@ func (client *Client) Snap(name string) (*Snap, error) {
 
 	return pkg, nil
 }
+
+// RemoveSnap removes the snap with the given name
+func (client *Client) RemoveSnap(name string) error {
+	const errPrefix = "cannot remove snap"
+	var rsp response
+
+	path := fmt.Sprintf("/2.0/snaps/%s", name)
+	body := strings.NewReader(`{"action":"remove"}`)
+
+	if err := client.do("POST", path, body, &rsp); err != nil {
```

Comment:
This looks like we want (eventually) to have a doAsync() method in the client to avoid duplication. But right now this is the only use of this (?) so its fine.